net.go

```diff
@@ -15,8 +16,8 @@ import (
 // there is not much port churn. Typically an OS will monotonically increase the
 // port numbers it assigns.
 func TryReserveRandomPort() (port int, err error) {
-	//nolint:gosec
-	listener, err := net.Listen("tcp", ":0")
+	var lc net.ListenConfig
```

Comment:
Are these listeners changing because of Go being backward breaking or the linter?